pyvista/core/utilities/reader.py

```diff
@@ -2831,6 +2839,261 @@ class ProStarReader(BaseReader):
     _vtk_class_name = 'vtkProStarReader'
 
 
+class ExodusIIReader(BaseReader, PointCellDataSelection, TimeReader):
+    """ExodusIIReader for .e and .exo files.
+
+    Reads Exodus II files
+
+    .. versionadded:: 0.45.0
+
+    Examples
+    --------
+    >>> import pyvista as pv
+    >>> from pyvista import examples
+    >>> filename = examples.download_mug(load=False)
+    >>> reader = pv.get_reader(filename)
+    >>> mesh = reader.read()
+    >>> mesh.plot()
+
+    """
+
+    _vtk_module_name = 'vtkIOExodus'
+    _vtk_class_name = 'vtkExodusIIReader'
+
+    def _set_defaults_post(self, filename):
+        super()._set_defaults_post(filename)
```

Comment:
This method doesn't accept a file name and the `super()` is a no-op, so we don't need to call it.  It is only available for Readers to modify the settings before reading the mesh.  `_set_defaults_post` sets the settings after reading the meta data from the file, which is appropriate here.

```suggestion
    def _set_defaults_post(self):
```